### ISSUE TITLE
The command doesn't work.

### ISSUE THREAD

**Issue by UCYT5040** [hallowed/calcboy#5]
Hi.

When I try to flash Calcboy I get this error:
```
make: *** No rule to make target 'epsilon_flash'.  Stop.
```
I've tried on multiple devices and it wont work.

I don't think that command exists anymore.

Please let me know if I need to provide additional information.

**Comment by lennardwalter** [comment]
hey
yeah i made this like two years ago.
don't really plan to fix it tbh, but you're welcome to open a pr or just use an old epsilon version

**Comment by UCYT5040** [comment]
Hey @lennardwalter,
It's really a bummer that you don't want to work on this anymore, but I understand.
I've tried using older epsilon versions but they don't seem to work.
Would you resume development for payment? I don't mind giving some money if you could get this working for me, on an old version or a new version.

Let me know if you are interested.

**Comment by lennardwalter** [comment]
nah, i don't want money for it... if you just want it to work (not necessarily with a new epsilon version), revert to this commit: `248d32920be9583cb4d0c14fbc0848987620ec18` (`git revert [hash]`).

works with `TARGET=simulator`, not sure if actual flashing also works, can't find my calculator.

lmk

**Comment by UCYT5040** [comment]
Hi, I'm getting all these errors and conflicts when trying to revert. I'm no Git professional, so is there something else I should be doing?

**Comment by lennardwalter** [comment]
`git clone https://github.com/numworks/epsilon`
`cd epsilon`
`git checkout 248d32920be9583cb4d0c14fbc0848987620ec18` (it's `checkout`, not `revert`, my bad)

then just do everything just like in the readme

**Comment by UCYT5040** [comment]
Hi, so sorry that I keep bothering you.
Now I get
```
ion/src/device/Makefile:4: ion/src/device/flasher/Makefile: No such file or directory
make: *** No rule to make target 'ion/src/device/flasher/Makefile'.  Stop.
```

I followed the same commands as you.

Again, I apologize for bothering you.
